src/custom/pages/Swap/SwapMod.tsx

```diff
@@ -204,7 +204,10 @@ export default function Swap({
   } = useDerivedSwapInfo()
 
   // detects trade load
-  const { quote, isGettingNewQuote } = useGetQuoteAndStatus({ token: INPUT.currencyId, chainId })
+  const { quote, isGettingNewQuote } = useGetQuoteAndStatus({
+    token: currencies.INPUT?.isNative ? currencies.INPUT.wrapped.address : INPUT.currencyId,
+    chainId,
```

Comment:
I didn't get why u had to do also this change, and not the one in `useCoingeckoUsdPrice`